TrafficReplayer/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.7.1-bin.zip
```

Comment:
Any issue with using newer version like 8.0.2? I don't know that this version will support Java 17 which the OS repo promotes